fetch-npm-node.js

```diff
@@ -1,16 +1,21 @@
 "use strict";
 
-var realFetch = require('node-fetch');
-module.exports = function(url, options) {
-	if (/^\/\//.test(url)) {
-		url = 'https:' + url;
-	}
-	return realFetch.call(this, url, options);
-};
-
-if (!global.fetch) {
-	global.fetch = module.exports;
-	global.Response = realFetch.Response;
-	global.Headers = realFetch.Headers;
-	global.Request = realFetch.Request;
+// add support for jsdomify for running tests
+// <https://github.com/niftylettuce/frisbee>
+// <https://github.com/podio/jsdomify>
+// <https://github.com/github/fetch/issues/238>
+if (typeof window === 'object') {
```

Comment:
check out my browser test on my package frisbee to see why - it doesn't
take into consideration for tests
On Nov 30, 2015 1:57 PM, "Matt Andrews" <notifications@github.com> wrote:

> In fetch-npm-node.js
> <https://github.com/matthew-andrews/isomorphic-fetch/pull/54#discussion_r46185792>
> :
>
> > -	if (/^\/\//.test(url)) {
> > -		url = 'https:' + url;
> > -	}
> > -	return realFetch.call(this, url, options);
> > -};
> > -
> > -if (!global.fetch) {
> > -	global.fetch = module.exports;
> > -	global.Response = realFetch.Response;
> > -	global.Headers = realFetch.Headers;
> > -	global.Request = realFetch.Request;
> > +// add support for jsdomify for running tests
> > +// <https://github.com/niftylettuce/frisbee>
> > +// <https://github.com/podio/jsdomify>
> > +// <https://github.com/github/fetch/issues/238>
> > +if (typeof window === 'object') {
>
> I don't understand … this file should never be required in a client side
> context … that's why the browser property is set … what am I missing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matthew-andrews/isomorphic-fetch/pull/54/files#r46185792>
> .
>